Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Bugzilla41271.cs

```diff
@@ -103,5 +121,24 @@ protected override void Init()
 				counter++;
 			}
 		}
+
+#if UITEST
+		// If the cellPhone has in Landscape mode, increase the timeout.
```

Comment:
Is this comment something we still need to do?

Reply:
No anymore, I forgot to delete that. Before I was scrolling until the 50ª cell, and in the landscape the test throw TimeOut Exception. Now I scroll until the 20ª cell